secure/storage/src/tests/suite.rs

```diff
@@ -34,16 +35,29 @@ const U64_KEY: &str = "U64_Key";
 const CRYPTO_NAME: &str = "Test_Key_Name";
 
 /// Executes all storage tests on a given storage backend.
-pub fn execute_all_storage_tests(storage: &mut BoxedStorage) {
+pub fn execute_all_storage_tests(storage: &mut Storage) {
     for test in STORAGE_TESTS.iter() {
         test(storage);
         storage.reset_and_clear().unwrap();
     }
 }
 
+/// This test tries to set a key, reset the storage and then retrieve its
```

Comment:
this is already implicitly tested throughout, but it seems alright to have an explicit test...